default.yml

```diff
@@ -70,9 +70,9 @@ Metrics/BlockLength:
 Metrics/MethodLength:
   Max: 20
 
-# The guiding principle of classes is SRP, SRP can't be accurately measured by LoC
+# SRP classes should fit in 400 lines. Use inline disable comments in god models.
 Metrics/ClassLength:
-  Max: 1500
+  Max: 400
```

Comment:
I will run this by dutchie ruby guild.

Meanwhile, if you are using this gem in GB you can override the setting here in a local `rubocop.yml`.